spec/helpers/pipeline_runs_helper_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+require "webmock/rspec"
+
+RSpec.describe PipelineRunsHelper, type: :helper do
+  describe "#new_failed_pipeline_run" do
+    def create_new_failed_pipeline_run(sample)
+      pr = helper.new_failed_pipeline_run(sample)
+      pr.error_message = "Failed Pipeline Run"
+      pr.save
+      return pr
+    end
+
+    before do
+      # Create a dummy alignment config for the pipeline run.
+      alignment_config = create(:alignment_config)
+      alignment_config.name = AlignmentConfig::DEFAULT_NAME
```

Comment:
if `AlignmentConfig::DEFAULT_NAME` is truly a default why doesn't the config object set it on creation?

Reply:
The naming isn't great.

It doesn't mean "the default name for a newly created alignment config", but rather, "the name of the default alignment config to use for newly created samples". Let me know if you have a suggestion for a better name (I can't think of one readily)